lib.typ

```diff
@@ -74,6 +113,7 @@
 }
 
 #let template(
+  lang: "en",
```

Comment:
Just a nitpick: I would put this argument a bit further down. Let's say between `AMS` and `heading-color`.